### PR TITLE
add missing Parishes of Bermuda (civicrm_state_province)

### PR DESCRIPTION
**Pull request by caracolmzd**
Overview
----------------------------------------
Add missing Parishes (State/Provinces) of Bermuda.

Please help if I added the incremental to the wrong file.

References:
 - http://statoids.com/ubm.html
 - https://www.bermudayp.com/article/view/1963///bermuda-postal-codes
 - https://www.bermudapost.bm/addressing/

### CODE REVIEW

**Comment by ginkgomzd** [comment]
Halp!
 Build failure:
 > No files found for pattern 'checkstyle/*.xml'. Configuration error?

File: xml/templates/civicrm_state_province.tpl [hunk 1]
```diff
@@ -4147,4 +4147,20 @@ INSERT INTO civicrm_state_province (id, country_id, abbreviation, name) VALUES
 (NULL, 1170, "DIN", "Dinagat Islands"),
 
 -- Add missing state for Colombia
-(NULL, 1048, "HUI", "Huila");
+(NULL, 1048, "HUI", "Huila"),
+
+-- Add missing parishes for Bermuda
+ (NULL, 'DEV', 'Devonshire'),
```
Comment (seamuslee001) [inline xml/templates/civicrm_state_province.tpl:4153]: @ginkgomzd the problem here is that your missing the country_id column (see above how it is the 2nd column)

Comment (ginkgomzd) [reply xml/templates/civicrm_state_province.tpl]: ah, thanks!

**Comment by ginkgomzd** [comment]
okay @demeritcowboy 

I moved the changes to the 5.50 incremental and ran regen.

Thanks

**Comment by demeritcowboy** [comment]
Thanks looks good.

**Pull request merged**